staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -268,7 +271,15 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		// on the size increase the "copy" operations in a json patch
 		// can cause.  See
 		// https://github.com/etcd-io/etcd/blob/release-3.3/etcdserver/server.go#L90.
-		JSONPatchMaxCopyBytes:        int64(10 * 1024 * 1024),
+		JSONPatchMaxCopyBytes: int64(10 * 1024 * 1024),
```

Comment:
I wan to loose the JSONPatchMaxCopyBytes the same in this pull. WDYT?

Reply:
yeah, probably so